server/routes.js

```diff
@@ -6,14 +6,33 @@
 
 import errors from './components/errors';
 import path from 'path';
+const authMiddlewares=require("./components/auth");
+const redirectUrlFunction = require("./components/auth/redirectUrl");
 
-export default function(app) {
+export default function (app) {
   // Insert routes below
-  app.use('/api/users', require('./api/thing'));
+  app.use('/api/users', require('./api/employee'));
+  //route for logout
+  app.use(authMiddlewares);
+  app.route("/logout")
+    .get((req, res) => {
+      res.clearCookie('Tsms').send("logout successfull");
+    });
+
+  app.route("/")
+    .get((req,res)=>{
+      res.send("hello everyone");
+    });
+  //Route to call redirect url from oAuth when sign in first time
+  app.route("/api/oauthServerCallback")
+    .get((req, res) => {
```

Comment:
Can be re-written in cleaner way:

app.route("/api/oauthServerCallback").get(redirectUrlFunction);
Or
app.get("/api/oauthServerCallback", redirectUrlFunction);